docs/topics/proc_debug-start-v8-debug-nodejs-openshift.adoc

```diff
@@ -0,0 +1,78 @@
+= Starting your Application on OpenShift and Attaching the V8 Inspector
+The V8 inspector enables you to debug your {NodeJS}{ndash}based application using other tools, such as link:https://developers.google.com/web/tools/chrome-devtools/[Chrome DevTools], that use the link:https://chromedevtools.github.io/debugger-protocol-viewer/1-2/[Chrome Debugging Protocol].
+
+.Prerequisites 
+* An application you want to debug with `nodeshift` installed.
+* The `oc` CLI client installed.
+* `npm` installed.
+* The V8 inspector installed, such as the one provided in link:https://www.google.com/chrome/index.html[the Google Chrome Browser].
+
+
+.Procedure
+. Log in to your OpenShift instance with the `oc` client.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Create a new project in OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc new-project {project-name}
+----
+
+. Use `nodeshift` to deploy the application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install && npm run openshift
+----
++
+The xref:about-nodeshift[Nodeshift module] is included with {NodeJS} boosters for deploying to OpenShift. If you are not using a booster, you can find more details on adding Nodeshift to your project in xref:proc_deploy-new-nodejs-app-openshift[].
+
+
+. List the deployment configuration.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get dc
+NAME          REVISION   DESIRED   CURRENT   TRIGGERED BY
+{app-name}   1          1         1         config,image({app-name}:latest)
+----
+
+. Enable debugging using an environment variable.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc set env dc/{app-name} NODE_ENV=development
+----
+
+. Check the status of your application and ensure your new pod is running.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get pods -w
+NAME                             READY     STATUS      RESTARTS   AGE
+{app-name}-2-aaaaa       1/1       Running     0          58s
+{app-name}-2-deploy   0/1       Completed   0          2m
+----
+
+. Forward the debugging port from the application running on OpenShift to `127.0.0.1:8888`.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc port-forward {app-name}-aaaaa 8888:5858
```

Comment:
Ah sorry, my bad -- the pod name here should be the same as in the `oc get pods -w` example: {app-name}-2-aaaaa

Reply:
@Ladicek nice catch, i completely missed that in copy/paste